README.md

```diff
@@ -16,109 +15,140 @@ The AdoptOpenJDK Build Farm is three things:
 * [openjdk-jdk9u](https://github.com/AdoptOpenJDK/openjdk-jdk9u)
 * [openjdk-jdk10u](https://github.com/AdoptOpenJDK/openjdk-jdk10u)
 * [openjdk-jdk11u](https://github.com/AdoptOpenJDK/openjdk-jdk11u)
 * [openjdk-jdk12u](https://github.com/AdoptOpenJDK/openjdk-jdk12u)
 * [openjdk-jdk13u](https://github.com/AdoptOpenJDK/openjdk-jdk13u)
+* [openjdk-jdk14u](https://github.com/AdoptOpenJDK/openjdk-jdk14u)
+* [openjdk-jdk15u](https://github.com/AdoptOpenJDK/openjdk-jdk15u)
+* [openjdk-jdk16](https://github.com/AdoptOpenJDK/openjdk-jdk16)
 * [openjdk-jdk](https://github.com/AdoptOpenJDK/openjdk-jdk)
 * [openjdk-jfx](https://github.com/AdoptOpenJDK/openjdk-jfx)
 * [openjdk-amber](https://github.com/AdoptOpenJDK/openjdk-amber)
+* [openjdk-amber-nightly](https://github.com/AdoptOpenJDK/openjdk-amber-nightly)
+* [openjdk-portola](https://github.com/AdoptOpenJDK/openjdk-portola)
+
+### AdoptOpenJDK binary repositories
+
+The following diagram lists the binary repositories for AdoptOpenJDK (edit the
+[Image Source](https://docs.google.com/drawings/d/1iJHzhLJ-cEL6mktWYzpWAYHHM0vOILT9ri4QYbsnGIg/edit) at Google and export to make changes.). These GitHub Repositories are where builds that pass the pipeline are published to. The API (and subsequently) website poll these repositories to make binaries available.
+
+![Distribution Repositories](images/Adopt_Build_Farm_Binary_Repos.png)
+
+* [openjdk8-binaries](https://github.com/AdoptOpenJDK/openjdk8-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 8 (HotSpot VM and Eclipse OpenJ9 VM).
+* [openjdk9-binaries](https://github.com/AdoptOpenJDK/openjdk9-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 9 (HotSpot VM and Eclipse OpenJ9 VM).
+* [openjdk10-binaries](https://github.com/AdoptOpenJDK/openjdk10-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 10 (HotSpot VM and Eclipse OpenJ9 VM).
+* [openjdk11-binaries](https://github.com/AdoptOpenJDK/openjdk11-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 11 (HotSpot VM and Eclipse OpenJ9 VM).
+* [openjdk12-binaries](https://github.com/AdoptOpenJDK/openjdk12-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 12 (HotSpot VM and Eclipse OpenJ9 VM).
+* [openjdk13-binaries](https://github.com/AdoptOpenJDK/openjdk13-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 13 (HotSpot VM and Eclipse OpenJ9 VM).
+* [openjdk14-binaries](https://github.com/AdoptOpenJDK/openjdk14-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 14 (HotSpot VM and Eclipse OpenJ9 VM).
+* [openjdk15-binaries](https://github.com/AdoptOpenJDK/openjdk15-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 15 (HotSpot VM and Eclipse OpenJ9 VM).
+* [openjdk16-binaries](https://github.com/AdoptOpenJDK/openjdk16-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 16 (HotSpot VM and Eclipse OpenJ9 VM).
```

Comment:
Include [openjdk17-binaries](https://github.com/AdoptOpenJDK/openjdk17-binaries/releases) too? I'd be slightly tempted to mark which of these are now obsolete and no longer updated, but is it really worth it